public_html/wp-content/plugins/camptix/camptix.php

```diff
@@ -3460,6 +3470,26 @@ public function get_segment( $relation, $conditions ) {
 				$post_query_conditions[] = $condition;
 				continue;
 			}
+
+			// Attended the event
+			if ( 'attended' == $condition['field'] ) {
```

Comment:
Again, this should use a strict comparison:
```suggestion
			if ( 'attended' === $condition['field'] ) {
```